operations/iuf/stages/deploy_product.md

```diff
@@ -17,18 +17,35 @@ The `deploy-product` stage changes the running state of the system.
 
 The following arguments are most often used with the `deploy-product` stage. See `iuf -h` and `iuf run -h` for additional arguments.
 
-| Input    | `iuf` Argument | Description                                            |
-|----------|----------------|--------------------------------------------------------|
-| Activity | `-a ACTIVITY`  | Activity created for the install or upgrade operations |
+| Input                                  | `iuf` Argument              | Description                                                                  |
+|----------------------------------------|-----------------------------|------------------------------------------------------------------------------|
+| Activity                               | `-a ACTIVITY`               | Activity created for the install or upgrade operations                       |
+| Site variables                         | `-sv SITE_VARS`             | Path to YAML file containing site defaults and any overrides                 |
+| Recipe variables                       | `-rv RECIPE_VARS`           | Path to YAML file containing recipe variables provided by HPE                |
+| `sat bootprep` configuration directory | `-bpcd BOOTPREP_CONFIG_DIR` | Directory containing `sat bootprep` configuration files and recipe variables |
 
 ## Execution details
 
-The code executed by this stage exists within IUF. See the `deploy-product` entry in `/usr/share/doc/csm/workflows/iuf/stages.yaml` and the corresponding files in `/usr/share/doc/csm/workflows/iuf/operations/` for details on the commands executed.
+The code executed by this stage exists within IUF. See the `deploy-product` entry in `/usr/share/doc/csm/workflows/iuf/stages.yaml`
+and the corresponding files in `/usr/share/doc/csm/workflows/iuf/operations/` for details on the commands executed.
 
 ## Example
 
 (`ncn-m001#`) Execute the `deploy-product` stage for activity `admin-230127`.
 
-```bash
-iuf -a admin-230127 run -r deploy-product
-```
+- Option 1: Install or upgrade without a workload manager
+
+  ```bash
+  iuf -a admin-230127 run -r deploy-product
+  ```
+
+- Option 2: Enable a workload manager with USS
+
+  When installing USS 1.1 or higher, select which workload manager to use on the system before running this stage. For more information, see the `deploy-product` stage
+  details in the "Install and Upgrade Framework" section of the
+  _HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)_.
```

Comment:
Lines 44-46 only apply to the `deliver-product` stage and can be removed here. There is no need to select the workload manager again.